package.json

```diff
@@ -1,83 +1,83 @@
 {
-    "name": "root",
-    "private": true,
-    "workspaces": [
-        "frontends/*"
-    ],
-    "scripts": {
-        "foreach": "yarn workspaces foreach -pvi --exclude=root run",
-        "start-dev": "yarn workspaces foreach -pvi run start-dev",
-        "build": "yarn workspaces foreach run build",
-        "lint-check": "yarn foreach global:lint-check",
-        "lint-fix": "yarn foreach global:lint-fix",
-        "fmt-check": "yarn foreach global:fmt-check",
-        "fmt-fix": "yarn foreach global:fmt-fix",
```

Comment:
Right `yarn fmt-check` (which we use on CI and in `./scripts/generate-openapi`) is only formatting JS/TS files.

I would suggest:
1. Change these two lines to `prettier --check .` and `prettier --write .`, respectively
    - I suspect this will also fix your CI issue... Now prettier will always run from the root, and should pick up `.prettierignore` and `.gitignore` automatically. [^1]
2. With (1), you should be able to clean out `.prettierignore`, except for `.yarn/` (which we want to commit, but we probably don't want to format).
3. Delete the `global:format-check` and `global:format-fix` commands below.

[^1]: I suspect it was working before because of some difference with running through prettier-eslint. :shrug: